---
assembly_status: <em style="color:forestgreen">Curated</em>
common_name: ''
data_arima-1_bases: 122.04 Gbp
data_arima-1_bytes: 64.1 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Aegotheles_albertisi/bAegAlb1/genomic_data/arima/
data_arima-1_scale: 1.7731
data_arima_bases: 122.04 Gbp
data_arima_bytes: 64.1 GiB
data_arima_coverage: N/A
data_pacbiohifi_bam-1_bases: 455.75 Gbp
data_pacbiohifi_bam-1_bytes: 200.6 GiB
data_pacbiohifi_bam-1_coverage: N/A
data_pacbiohifi_bam-1_links: s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_scale: 2.1162
data_pacbiohifi_bam_bases: 455.75 Gbp
data_pacbiohifi_bam_bytes: 200.6 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 50.42 Gbp
data_pacbiohifi_fqgz-1_bytes: 22.2 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 2.1162
data_pacbiohifi_fqgz_bases: 50.42 Gbp
data_pacbiohifi_fqgz_bytes: 22.2 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2024-04-25
hpa1filesize: 338.4 MiB
hpa1length: '1167479976'
hpa1n50ctg: 9620782
hpa1n50scf: 46898397
hpa1seq: https://s3.amazonaws.com/genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_curated/bAegAlb1.hap1.cur.20240425.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 6 </td><td> 17.49 Mbp </td><td> 2 </td><td> 84.73 Mbp </td></tr><tr><td> 20 </td><td> 13 </td><td> 15.91 Mbp </td><td> 3 </td><td> 84.22 Mbp </td></tr><tr><td> 30 </td><td> 21 </td><td> 14.23 Mbp </td><td> 4 </td><td> 81.92 Mbp </td></tr><tr><td> 40 </td><td> 30 </td><td> 11.70 Mbp </td><td> 6 </td><td> 59.52 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 41 </td><td style="background-color:#88ff88;"> 9.62 Mbp </td><td> 8 </td><td style="background-color:#88ff88;"> 46.90 Mbp </td></tr><tr><td> 60 </td><td> 55 </td><td> 6.07 Mbp </td><td> 11 </td><td> 41.67 Mbp </td></tr><tr><td> 70 </td><td> 77 </td><td> 4.72 Mbp </td><td> 14 </td><td> 27.45 Mbp </td></tr><tr><td> 80 </td><td> 107 </td><td> 2.88 Mbp </td><td> 19 </td><td> 20.78 Mbp </td></tr><tr><td> 90 </td><td> 168 </td><td> 1.22 Mbp </td><td> 25 </td><td> 15.13 Mbp </td></tr><tr><td> 100 </td><td> 1030 </td><td> 3.44 Kbp </td><td> 591 </td><td> 3.44 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1030 </th><th> 1.17 Gbp </th><th> 591 </th><th> 1.17 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_curated
hpb1date: 2024-05-06
hpb1filesize: 283.0 MiB
hpb1length: '1051698180'
hpb1n50ctg: 9598869
hpb1n50scf: 46606928
hpb1seq: https://s3.amazonaws.com/genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_curated/bAegAlb1.hap2.cur.20240506.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 5 </td><td> 21.94 Mbp </td><td> 1 </td><td> 110.21 Mbp </td></tr><tr><td> 20 </td><td> 10 </td><td> 17.34 Mbp </td><td> 3 </td><td> 82.07 Mbp </td></tr><tr><td> 30 </td><td> 17 </td><td> 14.86 Mbp </td><td> 4 </td><td> 81.20 Mbp </td></tr><tr><td> 40 </td><td> 24 </td><td> 13.13 Mbp </td><td> 6 </td><td> 57.10 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 34 </td><td style="background-color:#88ff88;"> 9.60 Mbp </td><td> 8 </td><td style="background-color:#88ff88;"> 46.61 Mbp </td></tr><tr><td> 60 </td><td> 47 </td><td> 7.03 Mbp </td><td> 10 </td><td> 42.05 Mbp </td></tr><tr><td> 70 </td><td> 63 </td><td> 5.36 Mbp </td><td> 13 </td><td> 27.36 Mbp </td></tr><tr><td> 80 </td><td> 86 </td><td> 3.94 Mbp </td><td> 17 </td><td> 20.68 Mbp </td></tr><tr><td> 90 </td><td> 129 </td><td> 1.41 Mbp </td><td> 24 </td><td> 12.81 Mbp </td></tr><tr><td> 100 </td><td> 869 </td><td> 4.48 Kbp </td><td> 525 </td><td> 4.48 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 869 </th><th> 1.05 Gbp </th><th> 525 </th><th> 1.05 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_curated
last_raw_data: 1711399894
last_updated: 1715016740
mds:
- data: species:&nbsp;Aegotheles&nbsp;albertisi<br>specimen:&nbsp;bAegAlb1<br>projects:&nbsp;<br>&nbsp;&nbsp;-&nbsp;vgp<br>assembled_by_group:&nbsp;Rockefeller<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>haplotype_to_curate:&nbsp;hap1<br>hap1:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/bAegAlb1.HiC.hap1.20231120.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/bAegAlb1.HiC.hap2.20231120.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bAegAlb1_hap1_s2.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bAegAlb1_hap2_s2.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/evaluation/merqury/bAegAlb1_png/<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/arima/<br>mito:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_MT_rockefeller/bAegAlb1.MT.20230105.fasta.gz<br>pipeline:&nbsp;<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.19.3+galaxy0)<br>&nbsp;&nbsp;-&nbsp;purge_dups&nbsp;(1.2.6+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy1)<br>&nbsp;&nbsp;-&nbsp;mitohifi&nbsp;(3+galaxy0)<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;Hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;bAegAlb1,&nbsp;resulting&nbsp;in&nbsp;two&nbsp;complete&nbsp;haplotypes.&nbsp;These&nbsp;assemblies&nbsp;underwent&nbsp;purge_dups.&nbsp;HiC&nbsp;scaffolding&nbsp;was&nbsp;performed&nbsp;with&nbsp;YaHS.&nbsp;&nbsp;The&nbsp;HiC&nbsp;prep&nbsp;kit&nbsp;used&nbsp;was&nbsp;Arima&nbsp;library&nbsp;prep.&nbsp;The&nbsp;HiC&nbsp;reads&nbsp;needed&nbsp;to&nbsp;have&nbsp;5&nbsp;bp&nbsp;trimmed&nbsp;from&nbsp;the&nbsp;5'&nbsp;end&nbsp;due&nbsp;to&nbsp;adapter&nbsp;left&nbsp;over&nbsp;from&nbsp;the&nbsp;Arima&nbsp;library&nbsp;prep&nbsp;kit.&nbsp;<br>
  ident: md10
  title: assembly_vgp_HiC_2.0/bAegAlb1_hap1.yml
- data: species:&nbsp;Aegotheles&nbsp;albertisi<br>specimen:&nbsp;bAegAlb1<br>projects:&nbsp;<br>&nbsp;&nbsp;-&nbsp;vgp<br>assembled_by_group:&nbsp;Rockefeller<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>haplotype_to_curate:&nbsp;hap2<br>hap1:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/bAegAlb1.HiC.hap1.20231120.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/bAegAlb1.HiC.hap2.20231120.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bAegAlb1_hap1_s2.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bAegAlb1_hap2_s2.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_vgp_HiC_2.0/evaluation/merqury/bAegAlb1_png/<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/genomic_data/arima/<br>mito:&nbsp;s3://genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_MT_rockefeller/bAegAlb1.MT.20230105.fasta.gz<br>pipeline:&nbsp;<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.19.3+galaxy0)<br>&nbsp;&nbsp;-&nbsp;purge_dups&nbsp;(1.2.6+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy1)<br>&nbsp;&nbsp;-&nbsp;mitohifi&nbsp;(3+galaxy0)<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;Hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;bAegAlb1,&nbsp;resulting&nbsp;in&nbsp;two&nbsp;complete&nbsp;haplotypes.&nbsp;These&nbsp;assemblies&nbsp;underwent&nbsp;purge_dups.&nbsp;HiC&nbsp;scaffolding&nbsp;was&nbsp;performed&nbsp;with&nbsp;YaHS.&nbsp;&nbsp;The&nbsp;HiC&nbsp;prep&nbsp;kit&nbsp;used&nbsp;was&nbsp;Arima&nbsp;library&nbsp;prep.&nbsp;The&nbsp;HiC&nbsp;reads&nbsp;needed&nbsp;to&nbsp;have&nbsp;5&nbsp;bp&nbsp;trimmed&nbsp;from&nbsp;the&nbsp;5'&nbsp;end&nbsp;due&nbsp;to&nbsp;adapter&nbsp;left&nbsp;over&nbsp;from&nbsp;the&nbsp;Arima&nbsp;library&nbsp;prep&nbsp;kit.&nbsp;<br>
  ident: md11
  title: assembly_vgp_HiC_2.0/bAegAlb1_hap2.yml
mito1date: 2024-01-05
mito1filesize: 5.4 KiB
mito1length: '17043'
mito1n50ctg: 17043
mito1n50scf: 17043
mito1seq: https://s3.amazonaws.com/genomeark/species/Aegotheles_albertisi/bAegAlb1/assembly_MT_rockefeller/bAegAlb1.MT.20240105.fasta.gz
mito1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 17.04 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 17.04 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 17.04 Kbp </td><td> 1 </td><td> 17.04 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 17.04 Kbp </th><th> 1 </th><th> 17.04 Kbp </th></tr>
  </tfoot>
  </table>
mito1version: assembly_MT_rockefeller
name: Aegotheles albertisi
name_: Aegotheles_albertisi
project: ~
short_name: bAegAlb
taxon_id: ''
---
