---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-1_bases: 586.19 Gbp
data_arima-1_bytes: 279.6 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/arima/
data_arima-1_scale: 1.9527
data_arima-3_bases: '586190495881'
data_arima-3_scale: '1.9527'
data_arima_bases: 586.19 Gbp
data_arima_bytes: 279.6 GiB
data_arima_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 119.52 Gbp
data_pacbiohifi_fqgz-1_bytes: 91.9 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.2115
data_pacbiohifi_fqgz-3_bases: '119522088294'
data_pacbiohifi_fqgz-3_scale: '1.2115'
data_pacbiohifi_fqgz_bases: 119.52 Gbp
data_pacbiohifi_fqgz_bytes: 91.9 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2023-08-15
hpa1filesize: 0.9 GiB
hpa1length: '3393661501'
hpa1n50ctg: 9691067
hpa1n50scf: 244821910
hpa1seq: https://s3.amazonaws.com/genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/aEleCoq1.HiC.hap1.20230815.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 7 </td><td> 35.38 Mbp </td><td> 1 </td><td> 0.54 Gbp </td></tr><tr><td> 20 </td><td> 19 </td><td> 23.03 Mbp </td><td> 2 </td><td> 346.26 Mbp </td></tr><tr><td> 30 </td><td> 36 </td><td> 18.50 Mbp </td><td> 3 </td><td> 314.08 Mbp </td></tr><tr><td> 40 </td><td> 59 </td><td> 12.62 Mbp </td><td> 4 </td><td> 313.53 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 90 </td><td style="background-color:#88ff88;"> 9.69 Mbp </td><td> 5 </td><td style="background-color:#88ff88;"> 244.82 Mbp </td></tr><tr><td> 60 </td><td> 129 </td><td> 7.78 Mbp </td><td> 7 </td><td> 221.97 Mbp </td></tr><tr><td> 70 </td><td> 183 </td><td> 5.23 Mbp </td><td> 8 </td><td> 202.38 Mbp </td></tr><tr><td> 80 </td><td> 265 </td><td> 3.38 Mbp </td><td> 10 </td><td> 143.70 Mbp </td></tr><tr><td> 90 </td><td> 403 </td><td> 1.72 Mbp </td><td> 13 </td><td> 126.40 Mbp </td></tr><tr><td> 100 </td><td> 1630 </td><td> 10.59 Kbp </td><td> 972 </td><td> 10.59 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1630 </th><th> 3.39 Gbp </th><th> 972 </th><th> 3.39 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_vgp_HiC_2.0
hpb1date: 2023-08-15
hpb1filesize: 0.9 GiB
hpb1length: '3340636055'
hpb1n50ctg: 10863754
hpb1n50scf: 277440397
hpb1seq: https://s3.amazonaws.com/genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/aEleCoq1.HiC.hap2.20230815.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 7 </td><td> 36.32 Mbp </td><td> 1 </td><td> 0.52 Gbp </td></tr><tr><td> 20 </td><td> 18 </td><td> 23.22 Mbp </td><td> 2 </td><td> 334.35 Mbp </td></tr><tr><td> 30 </td><td> 34 </td><td> 18.10 Mbp </td><td> 3 </td><td> 306.26 Mbp </td></tr><tr><td> 40 </td><td> 55 </td><td> 13.67 Mbp </td><td> 4 </td><td> 304.12 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 83 </td><td style="background-color:#88ff88;"> 10.86 Mbp </td><td> 5 </td><td style="background-color:#88ff88;"> 277.44 Mbp </td></tr><tr><td> 60 </td><td> 119 </td><td> 8.00 Mbp </td><td> 7 </td><td> 201.76 Mbp </td></tr><tr><td> 70 </td><td> 170 </td><td> 5.23 Mbp </td><td> 8 </td><td> 195.73 Mbp </td></tr><tr><td> 80 </td><td> 249 </td><td> 3.46 Mbp </td><td> 10 </td><td> 143.36 Mbp </td></tr><tr><td> 90 </td><td> 388 </td><td> 1.69 Mbp </td><td> 13 </td><td> 105.84 Mbp </td></tr><tr><td> 100 </td><td> 1510 </td><td> 9.38 Kbp </td><td> 854 </td><td> 9.38 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1510 </th><th> 3.34 Gbp </th><th> 854 </th><th> 3.34 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_vgp_HiC_2.0
last_raw_data: 1692138145
last_updated: 1692213998
mds:
- data: "species: Eleutherodactylus coqui\nspecimen: aEleCoq1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap1\nhap1: s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/aEleCoq1.HiC.hap1.20230815.fasta.gz\nhap2:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/aEleCoq1.HiC.hap2.20230815.fasta.gz\npretext_hap1:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/aEleCoq1_hap1_s2.pretext\npretext_hap2:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/aEleCoq1_hap2_s2.pretext\nkmer_spectra_img:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/evaluation/merqury/aEleCoq1_png/\nmito:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_MT_rockefeller/aEleCoq1.MT.20230815.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.19.3+galaxy0)\n  - yahs (1.2a.2)\nassembled_by_group: Rockefeller\nnotes:
    This was a hifiasm-HiC assembly of aEleCoq1 (VGL ID is aEleCoq3), resulting in
    two complete haplotypes. The provider listed the sample as male, and the kmer
    spectra seem to support a homogametic specimen. This individual did not bionano
    data. HiC scaffolding was performed with yahs. The HiC prep was Arima kit 2. The
    HiC reads needed to have 5 bp trimmed from the 5' end due to adapter left over
    from the Arima library prep kit. This is a phase 1 species. We are submitting
    both haplotypes for dual curation, and this is the ticket for hap1. "
  ident: md4
  title: assembly_vgp_HiC_2.0/aEleCoq1_hap1.yml
- data: "species: Eleutherodactylus coqui\nspecimen: aEleCoq1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap2\nhap1: s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/aEleCoq1.HiC.hap1.20230815.fasta.gz\nhap2:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/aEleCoq1.HiC.hap2.20230815.fasta.gz\npretext_hap1:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/aEleCoq1_hap1_s2.pretext\npretext_hap2:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/aEleCoq1_hap2_s2.pretext\nkmer_spectra_img:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_vgp_HiC_2.0/evaluation/merqury/aEleCoq1_png/\nmito:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_MT_rockefeller/aEleCoq1.MT.20230815.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Eleutherodactylus_coqui/aEleCoq1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.19.3+galaxy0)\n  - yahs (1.2a.2)\nassembled_by_group: Rockefeller\nnotes:
    This was a hifiasm-HiC assembly of aEleCoq1 (VGL ID is aEleCoq3), resulting in
    two complete haplotypes. The provider listed the sample as male, and the kmer
    spectra seem to support a homogametic specimen. This individual did not bionano
    data. HiC scaffolding was performed with yahs. The HiC prep was Arima kit 2. The
    HiC reads needed to have 5 bp trimmed from the 5' end due to adapter left over
    from the Arima library prep kit. This is a phase 1 species. We are submitting
    both haplotypes for dual curation, and this is the ticket for hap2. "
  ident: md5
  title: assembly_vgp_HiC_2.0/aEleCoq1_hap2.yml
mito1date: 2023-08-15
mito1filesize: 5.3 KiB
mito1length: '18519'
mito1n50ctg: 18519
mito1n50scf: 18519
mito1seq: https://s3.amazonaws.com/genomeark/species/Eleutherodactylus_coqui/aEleCoq1/assembly_MT_rockefeller/aEleCoq1.MT.20230815.fasta.gz
mito1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 18.52 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 18.52 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 18.52 Kbp </td><td> 1 </td><td> 18.52 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 18.52 Kbp </th><th> 1 </th><th> 18.52 Kbp </th></tr>
  </tfoot>
  </table>
mito1version: assembly_MT_rockefeller
name: Eleutherodactylus coqui
name_: Eleutherodactylus_coqui
project: ~
short_name: aEleCoq
taxon_id: ''
---
