---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-1_bases: 67.54 Gbp
data_arima-1_bytes: 35.0 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Heliangelus_exortis/bHelExo1/genomic_data/arima/
data_arima-1_scale: 1.7986
data_arima_bases: 67.54 Gbp
data_arima_bytes: 35.0 GiB
data_arima_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 48.08 Gbp
data_pacbiohifi_fqgz-1_bytes: 36.4 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Heliangelus_exortis/bHelExo1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.2284
data_pacbiohifi_fqgz_bases: 48.08 Gbp
data_pacbiohifi_fqgz_bytes: 36.4 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '''<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'''
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2023-04-19
hpa1filesize: 298.4 MiB
hpa1length: '1034859048'
hpa1n50ctg: 8479140
hpa1n50scf: 73778810
hpa1seq: https://s3.amazonaws.com/genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/bHelExo1.HiC.hap1.20230419.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 4 </td><td> 22.57 Mbp </td><td> 1 </td><td> 198.14 Mbp </td></tr><tr><td> 20 </td><td> 9 </td><td> 18.78 Mbp </td><td> 2 </td><td> 151.57 Mbp </td></tr><tr><td> 30 </td><td> 16 </td><td> 13.54 Mbp </td><td> 2 </td><td> 151.57 Mbp </td></tr><tr><td> 40 </td><td> 24 </td><td> 10.52 Mbp </td><td> 3 </td><td> 115.48 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 35 </td><td style="background-color:#88ff88;"> 8.48 Mbp </td><td> 4 </td><td style="background-color:#88ff88;"> 73.78 Mbp </td></tr><tr><td> 60 </td><td> 48 </td><td> 6.16 Mbp </td><td> 6 </td><td> 42.98 Mbp </td></tr><tr><td> 70 </td><td> 68 </td><td> 4.41 Mbp </td><td> 9 </td><td> 30.75 Mbp </td></tr><tr><td> 80 </td><td> 98 </td><td> 2.60 Mbp </td><td> 14 </td><td> 21.37 Mbp </td></tr><tr><td> 90 </td><td> 158 </td><td> 1.18 Mbp </td><td> 19 </td><td> 15.10 Mbp </td></tr><tr><td> 100 </td><td> 547 </td><td> 14.37 Kbp </td><td> 135 </td><td> 14.37 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 547 </th><th> 1.03 Gbp </th><th> 135 </th><th> 1.03 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_vgp_HiC_2.0
hpb1date: 2023-04-19
hpb1filesize: 280.9 MiB
hpb1length: '984688530'
hpb1n50ctg: 9824073
hpb1n50scf: 44470594
hpb1seq: https://s3.amazonaws.com/genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/bHelExo1.HiC.hap2.20230419.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 4 </td><td> 24.35 Mbp </td><td> 1 </td><td> 196.77 Mbp </td></tr><tr><td> 20 </td><td> 9 </td><td> 18.75 Mbp </td><td> 2 </td><td> 150.02 Mbp </td></tr><tr><td> 30 </td><td> 15 </td><td> 13.53 Mbp </td><td> 2 </td><td> 150.02 Mbp </td></tr><tr><td> 40 </td><td> 23 </td><td> 11.11 Mbp </td><td> 3 </td><td> 115.54 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 32 </td><td style="background-color:#88ff88;"> 9.82 Mbp </td><td> 4 </td><td style="background-color:#88ff88;"> 44.47 Mbp </td></tr><tr><td> 60 </td><td> 44 </td><td> 7.17 Mbp </td><td> 7 </td><td> 34.83 Mbp </td></tr><tr><td> 70 </td><td> 62 </td><td> 4.58 Mbp </td><td> 10 </td><td> 22.98 Mbp </td></tr><tr><td> 80 </td><td> 90 </td><td> 2.57 Mbp </td><td> 15 </td><td> 17.61 Mbp </td></tr><tr><td> 90 </td><td> 151 </td><td> 1.12 Mbp </td><td> 22 </td><td> 10.96 Mbp </td></tr><tr><td> 100 </td><td> 580 </td><td> 12.15 Kbp </td><td> 219 </td><td> 12.15 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 580 </th><th> 0.98 Gbp </th><th> 219 </th><th> 0.98 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_vgp_HiC_2.0
last_raw_data: 1681238907
last_updated: 1681939528
mds:
- data: "species: Heliangelus exortis\nspecimen: bHelExo1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap1\nhap1: s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/bHelExo1.HiC.hap1.20230419.fasta.gz\nhap2:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/bHelExo1.HiC.hap2.20230419.fasta.gz\npretext_hap1:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bHelExo1_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bHelExo1_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/evaluation/merqury_postpurge/bHelExo1_png/\npacbio_read_dir:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.19.3+galaxy0)\n  - purge_dups (1.2.6+galaxy0)\n  - yahs (1.2a.2+galaxy0)\nassembled_by_group:
    Rockefeller\nnotes: This was a hifiasm-HiC assembly of bHelExo1, resulting in
    two complete haplotypes. This individual did not have bionano data. HiC scaffolding
    was performed with yahs. K-mer spectra indicates a heterogametic specimen specimen.
    The HiC prep was Arima kit 2. I am submitting both hap1 & hap2 for dual curation.
    This is the curation ticket for hap1. "
  ident: md2
  title: assembly_vgp_HiC_2.0/bHelExo1_hap1.yml
- data: "species: Heliangelus exortis\nspecimen: bHelExo1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap2\nhap1: s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/bHelExo1.HiC.hap1.20230419.fasta.gz\nhap2:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/bHelExo1.HiC.hap2.20230419.fasta.gz\npretext_hap1:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bHelExo1_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bHelExo1_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/assembly_vgp_HiC_2.0/evaluation/merqury_postpurge/bHelExo1_png/\npacbio_read_dir:
    s3://genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Heliangelus_exortis/bHelExo1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.19.3+galaxy0)\n  - purge_dups (1.2.6+galaxy0)\n  - yahs (1.2a.2+galaxy0)\nassembled_by_group:
    Rockefeller\nnotes: This was a hifiasm-HiC assembly of bHelExo1, resulting in
    two complete haplotypes. This individual did not have bionano data. HiC scaffolding
    was performed with yahs. K-mer spectra indicates a heterogametic specimen specimen.
    The HiC prep was Arima kit 2. I am submitting both hap1 & hap2 for dual curation.
    This is the curation ticket for hap2. "
  ident: md3
  title: assembly_vgp_HiC_2.0/bHelExo1_hap2.yml
name: Heliangelus exortis
name_: Heliangelus_exortis
short_name: bHelExo
taxon_id: ''
---
