---
assembly_status: <em style="color:forestgreen">Curated</em>
common_name: ''
data_bionano-1_bases: N/A
data_bionano-1_bytes: 801.5 MiB
data_bionano-1_coverage: N/A
data_bionano-1_links: s3://genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/bionano/<br>
data_bionano-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/bionano/
data_bionano-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Kogia_breviceps/mKogBre1/genomic_data/bionano/
data_bionano-1_scale: 679.7017
data_bionano_bases: N/A
data_bionano_bytes: 801.5 MiB
data_bionano_coverage: N/A
data_pacbiohifi_bam-1_bases: 119.59 Gbp
data_pacbiohifi_bam-1_bytes: 408.7 GiB
data_pacbiohifi_bam-1_coverage: N/A
data_pacbiohifi_bam-1_links: s3://genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_scale: 0.2725
data_pacbiohifi_bam_bases: 119.59 Gbp
data_pacbiohifi_bam_bytes: 408.7 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_clr-1_bases: 463.83 Gbp
data_pacbiohifi_clr-1_bytes: 1.0 TiB
data_pacbiohifi_clr-1_coverage: N/A
data_pacbiohifi_clr-1_links: s3://genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_clr-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-1_scale: 0.4043
data_pacbiohifi_clr_bases: 463.83 Gbp
data_pacbiohifi_clr_bytes: 1.0 TiB
data_pacbiohifi_clr_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 119.84 Gbp
data_pacbiohifi_fqgz-1_bytes: 93.9 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Kogia_breviceps/mKogBre1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.1886
data_pacbiohifi_fqgz_bases: 119.84 Gbp
data_pacbiohifi_fqgz_bytes: 93.9 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '''<em style="color:forestgreen">PacBio HiFi</em>'''
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genbank_pri: mKogBre1:GCA_026419965.1 mKogBre1:GCA_026419985.1
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2022-11-03
hpa1filesize: 706.6 MiB
hpa1length: '2555838840'
hpa1n50ctg: 42780423
hpa1n50scf: 122394096
hpa1seq: https://s3.amazonaws.com/genomeark/species/Kogia_breviceps/mKogBre1/assembly_curated/mKogBre1.hap1.cur.20221103.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 122.12 Mbp </td><td> 1 </td><td> 201.99 Mbp </td></tr><tr><td> 20 </td><td> 4 </td><td> 69.84 Mbp </td><td> 2 </td><td> 188.80 Mbp </td></tr><tr><td> 30 </td><td> 8 </td><td> 53.58 Mbp </td><td> 3 </td><td> 184.93 Mbp </td></tr><tr><td> 40 </td><td> 13 </td><td> 47.07 Mbp </td><td> 5 </td><td> 147.15 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 19 </td><td style="background-color:#88ff88;"> 42.78 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 122.39 Mbp </td></tr><tr><td> 60 </td><td> 26 </td><td> 31.84 Mbp </td><td> 9 </td><td> 116.16 Mbp </td></tr><tr><td> 70 </td><td> 34 </td><td> 29.41 Mbp </td><td> 12 </td><td> 102.71 Mbp </td></tr><tr><td> 80 </td><td> 45 </td><td> 20.18 Mbp </td><td> 14 </td><td> 91.11 Mbp </td></tr><tr><td> 90 </td><td> 64 </td><td> 9.44 Mbp </td><td> 17 </td><td> 78.20 Mbp </td></tr><tr><td> 100 </td><td> 722 </td><td> 9.55 Kbp </td><td> 579 </td><td> 9.55 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 723 </th><th> 2.55 Gbp </th><th> 580 </th><th> 2.56 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_curated
hpb1date: 2022-11-02
hpb1filesize: 710.4 MiB
hpb1length: '2613416393'
hpb1n50ctg: 40421539
hpb1n50scf: 120411150
hpb1seq: https://s3.amazonaws.com/genomeark/species/Kogia_breviceps/mKogBre1/assembly_curated/mKogBre1.hap2.cur.20221102.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 2 </td><td> 92.56 Mbp </td><td> 1 </td><td> 195.90 Mbp </td></tr><tr><td> 20 </td><td> 4 </td><td> 86.54 Mbp </td><td> 2 </td><td> 191.84 Mbp </td></tr><tr><td> 30 </td><td> 8 </td><td> 57.81 Mbp </td><td> 4 </td><td> 157.65 Mbp </td></tr><tr><td> 40 </td><td> 13 </td><td> 49.77 Mbp </td><td> 5 </td><td> 146.29 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 19 </td><td style="background-color:#88ff88;"> 40.42 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 120.41 Mbp </td></tr><tr><td> 60 </td><td> 26 </td><td> 31.24 Mbp </td><td> 9 </td><td> 115.00 Mbp </td></tr><tr><td> 70 </td><td> 35 </td><td> 22.44 Mbp </td><td> 12 </td><td> 98.70 Mbp </td></tr><tr><td> 80 </td><td> 50 </td><td> 14.27 Mbp </td><td> 15 </td><td> 90.43 Mbp </td></tr><tr><td> 90 </td><td> 73 </td><td> 7.34 Mbp </td><td> 18 </td><td> 60.01 Mbp </td></tr><tr><td> 100 </td><td> 528 </td><td> 9.51 Kbp </td><td> 390 </td><td> 9.51 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 529 </th><th> 2.61 Gbp </th><th> 391 </th><th> 2.61 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_curated
last_raw_data: 1651598014
last_updated: 1667504411
mds:
- data: |+
    species: Kogia_breviceps
    specimen: mKogBre1
    projects:
      - vgp
    mat: s3://genomeark/species/Kogia_breviceps/mKogBre1/assembly_vgp_standard_2.0/mKogBre1.20220510.hap1.s2.fasta
    pat: s3://genomeark/species/Kogia_breviceps/mKogBre1/assembly_vgp_standard_2.0/mKogBre1.20220510.hap2.s2.fasta
    mat_hic_bam: s3://genomeark/species/Kogia_breviceps/mKogBre1/assembly_vgp_standard_2.0/evaluation/pretext/s2/mKogBre1.hap1.s2.bam
    pat_hic_bam: s3://genomeark/species/Kogia_breviceps/mKogBre1/assembly_vgp_standard_2.0/evaluation/pretext/s2/mKogBre1.hap2.s2.bam
    mat_pretext: s3://genomeark/species/Kogia_breviceps/mKogBre1/assembly_vgp_standard_2.0/evaluation/pretext/s2/mKogBre1.hap1.s2.pretext
    pat_pretext: s3://genomeark/species/Kogia_breviceps/mKogBre1/assembly_vgp_standard_2.0/evaluation/pretext/s2/mKogBre1.hap2.s2.pretext
    kmer_spectra_img: currently NA.
    pipeline:
      - hifiasm (0.16.1-r375)
      - solve (3.7)
      - salsa (2.3)


  ident: md4
  title: assembly_vgp_standard_2.0/mKogBre1.yaml
name: Kogia breviceps
name_: Kogia_breviceps
project: ~
short_name: mKogBre
taxon_id: ''
...
---
