---
assembly_status: <em style="color:maroon">No assembly</em>
common_name: ''
data_arima-2_bases: 242.39 Gbp
data_arima-2_bytes: 128.1 GiB
data_arima-2_coverage: N/A
data_arima-2_links: s3://genomeark/species/Latimeria_chalumnae/fLatCha2/genomic_data/arima/<br>
data_arima-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Latimeria_chalumnae/fLatCha2/genomic_data/arima/
data_arima-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Latimeria_chalumnae/fLatCha2/genomic_data/arima/
data_arima-2_scale: 1.7627
data_arima_bases: 242.39 Gbp
data_arima_bytes: 128.1 GiB
data_arima_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 123.61 Gbp
data_pacbiohifi_fqgz-1_bytes: 59.1 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Latimeria_chalumnae/fLatCha1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Latimeria_chalumnae/fLatCha1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Latimeria_chalumnae/fLatCha1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.9485
data_pacbiohifi_fqgz_bases: 123.61 Gbp
data_pacbiohifi_fqgz_bytes: 59.1 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
last_raw_data: 1702397217
last_updated: 1702680609
mds:
- data: "species: Latimeria chalumnae\nspecimen: fLatCha1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nprimary: s3://genomeark/species/Latimeria_chalumnae/fLatCha1/assembly_vgp_standard_2.0/fLatCha1.standard.pri.20231215.fasta.gz\nhaplotigs:
    s3://genomeark/species/Latimeria_chalumnae/fLatCha1/assembly_vgp_standard_2.0/fLatCha1.standard.alt.20231215.fasta.gz\npretext:
    s3://genomeark/species/Latimeria_chalumnae/fLatCha1/assembly_vgp_standard_2.0/evaluation/pri/pretext/fLatCha1_pri__s2.heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Latimeria_chalumnae/fLatCha1/assembly_vgp_standard_2.0/evaluation/merqury_purgedalt/fLatCha1_png/\nmito:
    s3://genomeark/species/Latimeria_chalumnae/fLatCha1/assembly_MT_rockefeller/fLatCha1.MT.20231215.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Latimeria_chalumnae/fLatCha1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Latimeria_chalumnae/fLatCha1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.18.8+galaxy1)\n  - yahs (1.2a.2+galaxy1)\nassembled_by_group: Rockefeller\nnotes:
    This was a primary/alternate assembly of fLatCha1. This is a VGP Phase 1 species.
    This individual did not have bionano data. HiC scaffolding was performed with
    yahs. The HiC prep was Arima kit 2. The HiC library kit was Arima. The HiC data
    came from a different individual, fLatCha2.  Purge_dups was used on the alternate
    assembly, but not on the primary assembly. The only purging on the primary assembly
    was Hifiasm's default purging when running in pseudohaplotype mode. "
  ident: md3
  title: assembly_vgp_standard_2.0/fLatCha1.yml
mito1date: 2023-12-15
mito1filesize: 5.2 KiB
mito1length: '16445'
mito1n50ctg: 16445
mito1n50scf: 16445
mito1seq: https://s3.amazonaws.com/genomeark/species/Latimeria_chalumnae/fLatCha1/assembly_MT_rockefeller/fLatCha1.MT.20231215.fasta.gz
mito1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 16.45 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 16.45 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 16.45 Kbp </td><td> 1 </td><td> 16.45 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 16.45 Kbp </th><th> 1 </th><th> 16.45 Kbp </th></tr>
  </tfoot>
  </table>
mito1version: assembly_MT_rockefeller
name: Latimeria chalumnae
name_: Latimeria_chalumnae
project: ~
short_name: fLatCha
taxon_id: ''
---
