---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-2_bases: 125.43 Gbp
data_arima-2_bytes: 61.5 GiB
data_arima-2_coverage: N/A
data_arima-2_links: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/<br>
data_arima-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/
data_arima-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Morus_bassanus/bMorBas2/genomic_data/arima/
data_arima-2_scale: 1.9002
data_arima_bases: 125.43 Gbp
data_arima_bytes: 61.5 GiB
data_arima_coverage: N/A
data_pacbiohifi_fqgz-2_bases: 49.62 Gbp
data_pacbiohifi_fqgz-2_bytes: 37.4 GiB
data_pacbiohifi_fqgz-2_coverage: N/A
data_pacbiohifi_fqgz-2_links: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-2_scale: 1.2354
data_pacbiohifi_fqgz_bases: 49.62 Gbp
data_pacbiohifi_fqgz_bytes: 37.4 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '''<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'''
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa2date: 2023-04-17
hpa2filesize: 391.6 MiB
hpa2length: '1380593596'
hpa2n50ctg: 9126423
hpa2n50scf: 66990878
hpa2seq: https://s3.amazonaws.com/genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap1.20230417.fasta.gz
hpa2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 5 </td><td> 27.27 Mbp </td><td> 1 </td><td> 168.77 Mbp </td></tr><tr><td> 20 </td><td> 11 </td><td> 18.70 Mbp </td><td> 2 </td><td> 134.71 Mbp </td></tr><tr><td> 30 </td><td> 19 </td><td> 16.88 Mbp </td><td> 3 </td><td> 116.62 Mbp </td></tr><tr><td> 40 </td><td> 29 </td><td> 12.21 Mbp </td><td> 5 </td><td> 85.50 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 42 </td><td style="background-color:#88ff88;"> 9.13 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 66.99 Mbp </td></tr><tr><td> 60 </td><td> 61 </td><td> 6.12 Mbp </td><td> 9 </td><td> 45.46 Mbp </td></tr><tr><td> 70 </td><td> 87 </td><td> 4.35 Mbp </td><td> 13 </td><td> 30.86 Mbp </td></tr><tr><td> 80 </td><td> 127 </td><td> 2.71 Mbp </td><td> 19 </td><td> 13.42 Mbp </td></tr><tr><td> 90 </td><td> 208 </td><td> 1.04 Mbp </td><td> 42 </td><td> 1.94 Mbp </td></tr><tr><td> 100 </td><td> 1175 </td><td> 10.74 Kbp </td><td> 935 </td><td> 10.74 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1175 </th><th> 1.38 Gbp </th><th> 935 </th><th> 1.38 Gbp </th></tr>
  </tfoot>
  </table>
hpa2version: assembly_vgp_HiC_2.0
hpb2date: 2023-04-17
hpb2filesize: 375.1 MiB
hpb2length: '1299462843'
hpb2n50ctg: 10178591
hpb2n50scf: 85624757
hpb2seq: https://s3.amazonaws.com/genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap2.20230417.fasta.gz
hpb2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 4 </td><td> 27.06 Mbp </td><td> 1 </td><td> 170.18 Mbp </td></tr><tr><td> 20 </td><td> 11 </td><td> 18.65 Mbp </td><td> 2 </td><td> 135.25 Mbp </td></tr><tr><td> 30 </td><td> 19 </td><td> 14.02 Mbp </td><td> 3 </td><td> 117.30 Mbp </td></tr><tr><td> 40 </td><td> 29 </td><td> 11.97 Mbp </td><td> 5 </td><td> 85.67 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 40 </td><td style="background-color:#88ff88;"> 10.18 Mbp </td><td> 6 </td><td style="background-color:#88ff88;"> 85.62 Mbp </td></tr><tr><td> 60 </td><td> 55 </td><td> 7.60 Mbp </td><td> 8 </td><td> 46.53 Mbp </td></tr><tr><td> 70 </td><td> 76 </td><td> 4.78 Mbp </td><td> 11 </td><td> 37.27 Mbp </td></tr><tr><td> 80 </td><td> 107 </td><td> 3.45 Mbp </td><td> 16 </td><td> 25.04 Mbp </td></tr><tr><td> 90 </td><td> 166 </td><td> 1.48 Mbp </td><td> 24 </td><td> 9.16 Mbp </td></tr><tr><td> 100 </td><td> 697 </td><td> 9.93 Kbp </td><td> 460 </td><td> 9.93 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 697 </th><th> 1.30 Gbp </th><th> 460 </th><th> 1.30 Gbp </th></tr>
  </tfoot>
  </table>
hpb2version: assembly_vgp_HiC_2.0
last_raw_data: 1677017682
last_updated: 1681781328
mds:
- data: "species: Morus bassanus\nspecimen: bMorBas2\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap1\nhap1: s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap1.20230417.fasta.gz\nhap2:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap2.20230417.fasta.gz\npretext_hap1:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bMorBas2_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bMorBas2_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/merqury/bMorBas2_png/\nmito:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_MT_rockefeller/bMorBas2.MT.20230411.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.19.3+galaxy0)\n  - yahs (1.2a.2+galaxy0)\nassembled_by_group: Rockefeller\nnotes:
    This was a hifiasm-HiC assembly of bMorBas2 (VGL ID is VGL-bMorBas1), resulting
    in two complete haplotypes. This individual did not have bionano data. HiC scaffolding
    was performed with yahs. K-mer spectra indicates a homogametic specimen. The HiC
    prep was Arima kit 2. I am submitting both hap1 & hap2 for dual curation. This
    is the curation ticket for hap1. "
  ident: md3
  title: assembly_vgp_HiC_2.0/bMorBas2_hap1.yml
- data: "species: Morus bassanus\nspecimen: bMorBas2\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap1\nhap1: s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap1.20230417.fasta.gz\nhap2:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap2.20230417.fasta.gz\npretext_hap1:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bMorBas2_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bMorBas2_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/merqury/bMorBas2_png/\nmito:
    s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_MT_rockefeller/bMorBas2.MT.20230411.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.19.3+galaxy0)\n  - yahs (1.2a.2+galaxy0)\nassembled_by_group: Rockefeller\nnotes:
    This was a hifiasm-HiC assembly of bMorBas2 (VGL ID is VGL-bMorBas1), resulting
    in two complete haplotypes. This individual did not have bionano data. HiC scaffolding
    was performed with yahs. K-mer spectra indicates a homogametic specimen. The HiC
    prep was Arima kit 2. I am submitting both hap1 & hap2 for dual curation. This
    is the curation ticket for hap1. "
  ident: md4
  title: assembly_vgp_HiC_2.0/bMorBas2_hap2.yml
mito2date: 2023-04-11
mito2filesize: 4.9 KiB
mito2length: '16690'
mito2n50ctg: 16690
mito2n50scf: 16690
mito2seq: https://s3.amazonaws.com/genomeark/species/Morus_bassanus/bMorBas2/assembly_MT_rockefeller/bMorBas2.MT.20230411.fasta.gz
mito2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 16.69 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 16.69 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 16.69 Kbp </th><th> 1 </th><th> 16.69 Kbp </th></tr>
  </tfoot>
  </table>
mito2version: assembly_MT_rockefeller
name: Morus bassanus
name_: Morus_bassanus
short_name: bMorBas
taxon_id: ''
---
