---
assembly_status: <em style="color:forestgreen">Curated</em>
common_name: ''
data_arima-1_bases: 121.26 Gbp
data_arima-1_bytes: 59.0 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/arima/
data_arima-1_scale: 1.9149
data_arima_bases: 121.26 Gbp
data_arima_bytes: 59.0 GiB
data_arima_coverage: N/A
data_pacbiohifi_clr-1_bases: 1145.38 Gbp
data_pacbiohifi_clr-1_bytes: 1.9 TiB
data_pacbiohifi_clr-1_coverage: N/A
data_pacbiohifi_clr-1_links: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_clr-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-1_scale: 0.5364
data_pacbiohifi_clr_bases: 1145.38 Gbp
data_pacbiohifi_clr_bytes: 1.9 TiB
data_pacbiohifi_clr_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 56.50 Gbp
data_pacbiohifi_fqgz-1_bytes: 44.2 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.1899
data_pacbiohifi_fqgz_bases: 56.50 Gbp
data_pacbiohifi_fqgz_bytes: 44.2 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2023-08-03
hpa1filesize: 397.4 MiB
hpa1length: '1482459898'
hpa1n50ctg: 14274197
hpa1n50scf: 80946297
hpa1seq: https://s3.amazonaws.com/genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_curated/bOpiHoa1.hap1.cur.20230803.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 3 </td><td> 40.42 Mbp </td><td> 1 </td><td> 157.52 Mbp </td></tr><tr><td> 20 </td><td> 8 </td><td> 26.46 Mbp </td><td> 3 </td><td> 109.48 Mbp </td></tr><tr><td> 30 </td><td> 14 </td><td> 21.99 Mbp </td><td> 4 </td><td> 96.05 Mbp </td></tr><tr><td> 40 </td><td> 21 </td><td> 18.59 Mbp </td><td> 6 </td><td> 88.57 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 30 </td><td style="background-color:#88ff88;"> 14.27 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 80.95 Mbp </td></tr><tr><td> 60 </td><td> 43 </td><td> 9.35 Mbp </td><td> 9 </td><td> 75.34 Mbp </td></tr><tr><td> 70 </td><td> 62 </td><td> 6.30 Mbp </td><td> 13 </td><td> 30.29 Mbp </td></tr><tr><td> 80 </td><td> 90 </td><td> 4.30 Mbp </td><td> 18 </td><td> 22.22 Mbp </td></tr><tr><td> 90 </td><td> 138 </td><td> 2.17 Mbp </td><td> 26 </td><td> 12.43 Mbp </td></tr><tr><td> 100 </td><td> 683 </td><td> 14.50 Kbp </td><td> 415 </td><td> 14.50 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 683 </th><th> 1.48 Gbp </th><th> 415 </th><th> 1.48 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_curated
hpb1date: 2023-08-03
hpb1filesize: 358.3 MiB
hpb1length: '1355639139'
hpb1n50ctg: 13957079
hpb1n50scf: 75689470
hpb1seq: https://s3.amazonaws.com/genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_curated/bOpiHoa1.hap2.cur.20230803.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 4 </td><td> 33.48 Mbp </td><td> 1 </td><td> 156.93 Mbp </td></tr><tr><td> 20 </td><td> 9 </td><td> 25.59 Mbp </td><td> 2 </td><td> 133.21 Mbp </td></tr><tr><td> 30 </td><td> 14 </td><td> 22.33 Mbp </td><td> 4 </td><td> 96.35 Mbp </td></tr><tr><td> 40 </td><td> 21 </td><td> 18.74 Mbp </td><td> 5 </td><td> 88.44 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 30 </td><td style="background-color:#88ff88;"> 13.96 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 75.69 Mbp </td></tr><tr><td> 60 </td><td> 40 </td><td> 11.16 Mbp </td><td> 8 </td><td> 74.87 Mbp </td></tr><tr><td> 70 </td><td> 55 </td><td> 7.66 Mbp </td><td> 12 </td><td> 30.14 Mbp </td></tr><tr><td> 80 </td><td> 77 </td><td> 4.60 Mbp </td><td> 18 </td><td> 19.43 Mbp </td></tr><tr><td> 90 </td><td> 120 </td><td> 2.07 Mbp </td><td> 27 </td><td> 9.91 Mbp </td></tr><tr><td> 100 </td><td> 535 </td><td> 15.49 Kbp </td><td> 323 </td><td> 15.49 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 535 </th><th> 1.36 Gbp </th><th> 323 </th><th> 1.36 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_curated
last_raw_data: 1674520386
last_updated: 1691021004
mds:
- data: "species: Opisthocomus hoazin\nspecimen: bOpiHoa1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap1\nhap1: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/bOpiHoa1.HiC.hap1.20230227.fasta.gz\nhap2:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/bOpiHoa1.HiC.hap2.20230227.fasta.gz\npretext_hap1:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bOpiHoa1_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bOpiHoa1_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/evaluation/merqury/bOpiHoa1_png/\nmito:
    s3://genomeark/working/temp_mitos/bOpiHoa1_mitohifi_final.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.16.1+galaxy4)\n  - yahs (1.2a.2+galaxy0)\nassembled_by_group: Rockefeller\nnotes:
    This was a hifiasm-HiC assembly of bOpiHoa1, resulting in two complete haplotypes.
    This individual did not have bionano data. HiC scaffolding was performed with
    yahs. The HiC prep was Arima kit 2. The kmer spectra indicates a heterogametic
    specimen, so we are submitting both haplotypes as tickets, in case the sex chromosomes
    need to be fished out between the haplotypes. This is the curation ticket for
    hap1. MitoHiFi was run on the raw reads, and the resulting mitogenome is included
    in this ticket for help with mitochondrial decontam, but the final mitogenome
    we upload may differ as we want to verify the presence of a large repeat in the
    mito assembly."
  ident: md8
  title: assembly_vgp_HiC_2.0/bOpiHoa1_hap1.yml
- data: "species: Opisthocomus hoazin\nspecimen: bOpiHoa1\nprojects: \n  - vgp\ndata_location:
    S3\nrelease_to: S3\nhaplotype_to_curate: hap2\nhap1: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/bOpiHoa1.HiC.hap1.20230227.fasta.gz\nhap2:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/bOpiHoa1.HiC.hap2.20230227.fasta.gz\npretext_hap1:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bOpiHoa1_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bOpiHoa1_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/assembly_vgp_HiC_2.0/evaluation/merqury/bOpiHoa1_png/\nmito:
    s3://genomeark/working/temp_mitos/bOpiHoa1_mitohifi_final.fasta.gz\npacbio_read_dir:
    s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Opisthocomus_hoazin/bOpiHoa1/genomic_data/arima/\npipeline:\n
    \ - hifiasm (0.16.1+galaxy4)\n  - yahs (1.2a.2+galaxy0)\nassembled_by_group: Rockefeller\nnotes:
    This was a hifiasm-HiC assembly of bOpiHoa1, resulting in two complete haplotypes.
    This individual did not have bionano data. HiC scaffolding was performed with
    yahs. The HiC prep was Arima kit 2. The kmer spectra indicates a heterogametic
    specimen, so we are submitting both haplotypes as tickets, in case the sex chromosomes
    need to be fished out between the haplotypes. This is the curation ticket for
    hap2. MitoHiFi was run on the raw reads, and the resulting mitogenome is included
    in this ticket for help with mitochondrial decontam, but the final mitogenome
    we upload may differ as we want to verify the presence of a large repeat in the
    mito assembly."
  ident: md9
  title: assembly_vgp_HiC_2.0/bOpiHoa1_hap2.yml
name: Opisthocomus hoazin
name_: Opisthocomus_hoazin
project: ~
short_name: bOpiHoa
taxon_id: ''
---
