---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_dovetail-1_bases: 200.17 Gbp
data_dovetail-1_bytes: 99.6 GiB
data_dovetail-1_coverage: N/A
data_dovetail-1_links: s3://genomeark/species/Passer_domesticus/bPasDom1/genomic_data/dovetail/<br>
data_dovetail-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Passer_domesticus/bPasDom1/genomic_data/dovetail/
data_dovetail-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Passer_domesticus/bPasDom1/genomic_data/dovetail/
data_dovetail-1_scale: 1.8711
data_dovetail_bases: 200.17 Gbp
data_dovetail_bytes: 99.6 GiB
data_dovetail_coverage: N/A
data_pacbiohifi_clr-1_bases: 454.14 Gbp
data_pacbiohifi_clr-1_bytes: 790.1 GiB
data_pacbiohifi_clr-1_coverage: N/A
data_pacbiohifi_clr-1_links: s3://genomeark/species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_clr-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-1_scale: 0.5353
data_pacbiohifi_clr_bases: 454.14 Gbp
data_pacbiohifi_clr_bytes: 790.1 GiB
data_pacbiohifi_clr_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 61.54 Gbp
data_pacbiohifi_fqgz-1_bytes: 54.0 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.0615
data_pacbiohifi_fqgz_bases: 61.54 Gbp
data_pacbiohifi_fqgz_bytes: 54.0 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '''<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Dovetail</em>'''
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2023-05-10
hpa1filesize: 321.2 MiB
hpa1length: '1204565223'
hpa1n50ctg: 21374955
hpa1n50scf: 68955217
hpa1seq: https://s3.amazonaws.com/genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/bPasDom1.HiC.hap1.20230510.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 2 </td><td> 55.49 Mbp </td><td> 1 </td><td> 161.09 Mbp </td></tr><tr><td> 20 </td><td> 4 </td><td> 50.58 Mbp </td><td> 2 </td><td> 116.02 Mbp </td></tr><tr><td> 30 </td><td> 7 </td><td> 36.42 Mbp </td><td> 3 </td><td> 115.61 Mbp </td></tr><tr><td> 40 </td><td> 11 </td><td> 31.37 Mbp </td><td> 5 </td><td> 82.48 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 15 </td><td style="background-color:#88ff88;"> 21.37 Mbp </td><td> 6 </td><td style="background-color:#88ff88;"> 68.96 Mbp </td></tr><tr><td> 60 </td><td> 21 </td><td> 17.36 Mbp </td><td> 8 </td><td> 43.45 Mbp </td></tr><tr><td> 70 </td><td> 29 </td><td> 13.71 Mbp </td><td> 12 </td><td> 23.72 Mbp </td></tr><tr><td> 80 </td><td> 40 </td><td> 7.93 Mbp </td><td> 18 </td><td> 13.98 Mbp </td></tr><tr><td> 90 </td><td> 65 </td><td> 2.78 Mbp </td><td> 30 </td><td> 6.71 Mbp </td></tr><tr><td> 100 </td><td> 518 </td><td> 2.00 Kbp </td><td> 379 </td><td> 2.00 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 518 </th><th> 1.20 Gbp </th><th> 379 </th><th> 1.20 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_vgp_HiC_2.0
hpb1date: 2023-05-10
hpb1filesize: 363.9 MiB
hpb1length: '1364778654'
hpb1n50ctg: 14787547
hpb1n50scf: 74196017
hpb1seq: https://s3.amazonaws.com/genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/bPasDom1.HiC.hap2.20230510.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 3 </td><td> 48.11 Mbp </td><td> 1 </td><td> 162.12 Mbp </td></tr><tr><td> 20 </td><td> 6 </td><td> 40.56 Mbp </td><td> 2 </td><td> 121.33 Mbp </td></tr><tr><td> 30 </td><td> 11 </td><td> 26.12 Mbp </td><td> 4 </td><td> 82.21 Mbp </td></tr><tr><td> 40 </td><td> 16 </td><td> 21.74 Mbp </td><td> 5 </td><td> 78.34 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 24 </td><td style="background-color:#88ff88;"> 14.79 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 74.20 Mbp </td></tr><tr><td> 60 </td><td> 34 </td><td> 11.66 Mbp </td><td> 10 </td><td> 35.98 Mbp </td></tr><tr><td> 70 </td><td> 49 </td><td> 7.48 Mbp </td><td> 15 </td><td> 20.43 Mbp </td></tr><tr><td> 80 </td><td> 71 </td><td> 4.96 Mbp </td><td> 26 </td><td> 8.37 Mbp </td></tr><tr><td> 90 </td><td> 114 </td><td> 1.97 Mbp </td><td> 46 </td><td> 4.52 Mbp </td></tr><tr><td> 100 </td><td> 681 </td><td> 1.00 Kbp </td><td> 517 </td><td> 1.00 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 681 </th><th> 1.36 Gbp </th><th> 517 </th><th> 1.36 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_vgp_HiC_2.0
last_raw_data: 1684351286
last_updated: 1684351286
mds:
- data: "species: Passer domesticus\nspecimen: bPasDom1\nprojects:\n  - erga       \n
    \ - vgp\nhap1: s3://genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/bPasDom1.HiC.hap1.20230510.fasta.gz\nhap2:
    s3://genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/bPasDom1.HiC.hap2.20230510.fasta.gz\nprimary:
    \npretext_hap1: s3://genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bPasDom1_hap1_s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bPasDom1_hap2_s2_heatmap.pretext\npretext_primary:\nkmer_spectra_img:
    s3://genomeark/species/Passer_domesticus/bPasDom1/assembly_vgp_HiC_2.0/evaluation/merqury/bPasDom1_png/\npacbio_read_dir:
    s3://genomeark/species/Passer_domesticus/bPasDom1/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Passer_domesticus/bPasDom1/genomic_data/dovetail/\nhic_kit:
    Dovetail OmniC\npipeline:\n  - hifiasm (0.19.3-r575)\n  - yahs (1.2a.1)\nnotes:
    This was a hifiasm-HiC assembly of bPasDom1, resulting in two complete haplotypes.
    This assembly was done by Gabriel David, Uppsala university, Sweden. contact:
    gbdnorden@gmail.com\n"
  ident: md2
  title: assembly_vgp_HiC_2.0/bPasDom1.yaml
name: Passer domesticus
name_: Passer_domesticus
short_name: bPasDom
taxon_id: ''
---
