---
alt3date: 2022-12-17
alt3filesize: 158.3 KiB
alt3length: '543247'
alt3n50ctg: 0
alt3n50scf: 0
alt3seq: https://s3.amazonaws.com/genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_curated/aPseCor3.alt.cur.20221217.fasta.gz
alt3sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 20 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 30 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 40 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 60 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 70 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 80 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 90 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 100 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr></tbody>
  <tfoot>
  <tr><th> 0.000x </th><th> 1 </th><th> 0.54 Mbp </th><th> 1 </th><th> 0.54 Mbp </th></tr>
  </tfoot>
  </table>
alt3version: assembly_curated
assembly_status: <em style="color:forestgreen">Curated</em>
common_name: corroboree frog
data_arima-3_bases: 381.02 Gbp
data_arima-3_bytes: 188.4 GiB
data_arima-3_coverage: 44.53x
data_arima-3_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/arima/<br>
data_arima-3_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/arima/
data_arima-3_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor3/genomic_data/arima/
data_arima-3_scale: 1.8836
data_arima_bases: 381.02 Gbp
data_arima_bytes: 188.4 GiB
data_arima_coverage: 44.53x
data_bionano-3_bases: N/A
data_bionano-3_bytes: 17.4 GiB
data_bionano-3_coverage: 1241.01x
data_bionano-3_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/bionano/<br>
data_bionano-3_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/bionano/
data_bionano-3_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor3/genomic_data/bionano/
data_bionano-3_scale: 570.0040
data_bionano_bases: N/A
data_bionano_bytes: 17.4 GiB
data_bionano_coverage: 1241.01x
data_illumina-1_bases: 387.72 Gbp
data_illumina-1_bytes: 184.5 GiB
data_illumina-1_coverage: 45.31x
data_illumina-1_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor1/genomic_data/illumina/<br>
data_illumina-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor1/genomic_data/illumina/
data_illumina-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor1/genomic_data/illumina/
data_illumina-1_scale: 1.9568
data_illumina-2_bases: 366.86 Gbp
data_illumina-2_bytes: 174.0 GiB
data_illumina-2_coverage: 42.87x
data_illumina-2_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor2/genomic_data/illumina/<br>
data_illumina-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor2/genomic_data/illumina/
data_illumina-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor2/genomic_data/illumina/
data_illumina-2_scale: 1.9635
data_illumina_bases: 754.58 Gbp
data_illumina_bytes: 358.5 GiB
data_illumina_coverage: 88.18x
data_pacbiohifi_bam-3_bases: 229.32 Gbp
data_pacbiohifi_bam-3_bytes: 788.4 GiB
data_pacbiohifi_bam-3_coverage: 26.80x
data_pacbiohifi_bam-3_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-3_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-3_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-3_scale: 0.2709
data_pacbiohifi_bam_bases: 229.32 Gbp
data_pacbiohifi_bam_bytes: 788.4 GiB
data_pacbiohifi_bam_coverage: 26.80x
data_pacbiohifi_clr-3_bases: 820.24 Gbp
data_pacbiohifi_clr-3_bytes: 2.0 TiB
data_pacbiohifi_clr-3_coverage: 95.86x
data_pacbiohifi_clr-3_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_clr-3_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-3_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-3_scale: 0.3821
data_pacbiohifi_clr_bases: 820.24 Gbp
data_pacbiohifi_clr_bytes: 2.0 TiB
data_pacbiohifi_clr_coverage: 95.86x
data_pacbiohifi_fqgz-3_bases: 230.31 Gbp
data_pacbiohifi_fqgz-3_bytes: 175.0 GiB
data_pacbiohifi_fqgz-3_coverage: 26.91x
data_pacbiohifi_fqgz-3_links: s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-3_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-3_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-3_scale: 1.2259
data_pacbiohifi_fqgz_bases: 230.31 Gbp
data_pacbiohifi_fqgz_bytes: 175.0 GiB
data_pacbiohifi_fqgz_coverage: 26.91x
data_status: '''<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>
  ::: <em style="color:forestgreen">Illumina</em>'''
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genbank_pri: aPseCor3:GCA_028390025.1 aPseCor3:GCA_028390055.1
genome_size: 8557000000
genome_size_display: 8.56 Gbp
genome_size_method: GoAT
hpa3date: 2022-08-06
hpa3filesize: 2.3 GiB
hpa3length: '8945066269'
hpa3n50ctg: 7191000
hpa3n50scf: 679934666
hpa3seq: https://s3.amazonaws.com/genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_vgp_HiC_2.0/aPseCor3.HiC.hap1.20220806.fasta.gz
hpa3sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 31 </td><td> 22.87 Mbp </td><td> 1 </td><td> 1.22 Gbp </td></tr><tr><td> 20 </td><td> 77 </td><td> 15.59 Mbp </td><td> 2 </td><td> 1.04 Gbp </td></tr><tr><td> 30 </td><td> 141 </td><td> 11.71 Mbp </td><td> 3 </td><td> 0.93 Gbp </td></tr><tr><td> 40 </td><td> 224 </td><td> 9.09 Mbp </td><td> 4 </td><td> 0.82 Gbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 330 </td><td style="background-color:#88ff88;"> 7.19 Mbp </td><td> 5 </td><td style="background-color:#88ff88;"> 0.68 Gbp </td></tr><tr><td> 60 </td><td> 467 </td><td> 5.40 Mbp </td><td> 6 </td><td> 0.66 Gbp </td></tr><tr><td> 70 </td><td> 655 </td><td> 3.87 Mbp </td><td> 8 </td><td> 466.06 Mbp </td></tr><tr><td> 80 </td><td> 927 </td><td> 2.52 Mbp </td><td> 11 </td><td> 206.57 Mbp </td></tr><tr><td> 90 </td><td> 1386 </td><td> 1.36 Mbp </td><td> 16 </td><td> 76.73 Mbp </td></tr><tr><td> 100 </td><td> 2557 </td><td> 361.00 Kbp </td><td> 424 </td><td> 0.59 Mbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.043x </th><th> 5957 </th><th> 8.93 Gbp </th><th> 3526 </th><th> 8.95 Gbp </th></tr>
  </tfoot>
  </table>
hpa3version: assembly_vgp_HiC_2.0
hpb3date: 2022-12-17
hpb3filesize: 2.3 GiB
hpb3length: '8872758793'
hpb3n50ctg: 7097000
hpb3n50scf: 846894890
hpb3seq: https://s3.amazonaws.com/genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_curated/aPseCor3.hap2.cur.20221217.fasta.gz
hpb3sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 28 </td><td> 22.40 Mbp </td><td> 0 </td><td> 1.25 Gbp </td></tr><tr><td> 20 </td><td> 75 </td><td> 15.29 Mbp </td><td> 1 </td><td> 1.06 Gbp </td></tr><tr><td> 30 </td><td> 141 </td><td> 11.59 Mbp </td><td> 2 </td><td> 0.95 Gbp </td></tr><tr><td> 40 </td><td> 224 </td><td> 9.09 Mbp </td><td> 3 </td><td> 0.85 Gbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 330 </td><td style="background-color:#88ff88;"> 7.10 Mbp </td><td> 4 </td><td style="background-color:#88ff88;"> 0.85 Gbp </td></tr><tr><td> 60 </td><td> 469 </td><td> 5.22 Mbp </td><td> 5 </td><td> 0.81 Gbp </td></tr><tr><td> 70 </td><td> 670 </td><td> 3.60 Mbp </td><td> 6 </td><td> 0.52 Gbp </td></tr><tr><td> 80 </td><td> 964 </td><td> 2.31 Mbp </td><td> 8 </td><td> 483.89 Mbp </td></tr><tr><td> 90 </td><td> 1474 </td><td> 1.18 Mbp </td><td> 10 </td><td> 366.57 Mbp </td></tr><tr><td> 100 </td><td> 2906 </td><td> 268.00 Kbp </td><td> 408 </td><td> 412.31 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.034x </th><th> 5826 </th><th> 8.85 Gbp </th><th> 3127 </th><th> 8.87 Gbp </th></tr>
  </tfoot>
  </table>
hpb3version: assembly_curated
last_raw_data: 1678825205
last_updated: 1678825205
mds:
- data: "species: Pseudophryne corroboree\nspecimen: aPseCor3\nprojects: \n  - vgp\nhap1:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_vgp_HiC_2.0/aPseCor3.HiC.hap1.20220806.fasta.gz\nhap2:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_vgp_HiC_2.0/aPseCor3.HiC.hap2.20220806.fasta.gz\npretext_hap1:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/aPseCor3_hap1__s2_heatmap.pretext\npretext_hap2:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/aPseCor3_hap2__s2_heatmap.pretext\nkmer_spectra_img:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/assembly_vgp_HiC_2.0/evaluation/aPseCor3_png/\npacbio_read_dir:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/pacbio_hifi/\npacbio_read_type:
    hifi\nhic_read_dir: s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/arima/\nbionano_cmap_dir:
    s3://genomeark/species/Pseudophryne_corroboree/aPseCor3/genomic_data/bionano/\npipeline:\n
    \ - hifiasm (0.16.1+galaxy2)\n  - solve (3.7.0+galaxy0)\n  - yahs (1.2a+galaxy1)\nassembled_by_group:
    Rockefeller\nnotes: Please curate hap2 ( aPseCor3.HiC.hap2.20220806.fasta.gz ),
    as it looks like the more complete assembly between the two of them. I am including
    the hap1 paths here in case they would be useful. "
  ident: md8
  title: assembly_vgp_HiC_2.0/aPseCor3_hap2.yaml
metadata: |
  species:
    name: Pseudophryne corroboree
    individuals:
    - short_name: aPseCor3
    short_name: aPseCor
    taxon_id: 495146
    common_name: corroboree frog
    genome_size: 8557000000
    genome_size_method: GoAT
    order:
      name: Anura
    family:
      name: Myobatrachidae
    project: [ vgp ]
name: Pseudophryne corroboree
name_: Pseudophryne_corroboree
project:
- vgp
short_name: aPseCor
taxon_id: 495146
---
