---
layout: default
---

{% comment %} This sets the 'image' and 'license' variables used below {% endcomment %}
{% include species-image.html species_name=page.name_ %}

<div class="section">
<div>
<div class="sd-info">
  <p class="sd-sname">{{ page.name }}</p>
  <p class="sd-cname">{{ page.common_name }}</p>

  <br>

  {% if page.taxon_id %}
    <table class="sd-info-tab">
    <thead><tr><th class="label" width="30%">Resource</th>
               <th></th>
           </tr></thead>
    <tbody><tr><td class="label">NCBI Taxonomy Browser</td>
               <td><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id={{ page.taxon_id }}">NCBI:txid{{ page.taxon_id }}</a></td>
           </tr>
           <tr><td class="label">S3 HTML Index</td>
               <td><a href="https://genomeark.s3.amazonaws.com/index.html?prefix=species/{{ page.name_ }}">{{ page.name }}</a></td>
           </tr>
           <tr><td class="label">S3 GUI Browser</td>
               <td><a href="https://42basepairs.com/browse/s3/genomeark/species/{{ page.name_ }}">{{ page.name }}</a></td>
           </tr></tbody>
    </table>
    <br>
  {% endif %}

  {% if page.genbank_pri or page.genbank_alt %}
    <table class="sd-info-tab">
    <thead><tr><th class="label" width="30%">Resource</th>
               <th>Primary Assembly</th>
               <th>Alternate Assembly</th>
           </tr></thead>
    <tbody><tr><td class="label">GenBank</td>
               <td> {% include resource-links.html data=page.genbank_pri link="https://www.ncbi.nlm.nih.gov/assembly/YOURACCHERE" %} </td>
               <td> {% include resource-links.html data=page.genbank_alt link="https://www.ncbi.nlm.nih.gov/assembly/YOURACCHERE" %} </td>
           </tr>
           <tr><td class="label">ENA</td>
               <td> {% include resource-links.html data=page.genbank_pri link="https://www.ebi.ac.uk/ena/browser/view/YOURACCHERE" %} </td>
               <td> {% include resource-links.html data=page.genbank_alt link="https://www.ebi.ac.uk/ena/browser/view/YOURACCHERE" %} </td>
           </tr>
           <tr><td class="label">UCSC Genome Browser</td>
               <td> {% include resource-links.html data=page.genbank_pri link="https://genome.ucsc.edu/h/YOURACCHERE" %} </td>
               <td> {% include resource-links.html data=page.genbank_alt link="https://genome.ucsc.edu/h/YOURACCHERE" %} </td>
           </tr></tbody>
    </table>
    <br>
  {% endif %}

  {% if page.genbank_mat or page.genbank_pat %}
    <table class="sd-info-tab">
    <thead><tr><th class="label" width="30%">Resource</th>
               <th>Maternal Assembly</th>
               <th>Paternal Assembly</th>
           </tr></thead>
    <tbody><tr><td class="label">GenBank</td>
               <td> {% include resource-links.html data=page.genbank_mat link="https://www.ncbi.nlm.nih.gov/assembly/YOURACCHERE" %} </td>
               <td> {% include resource-links.html data=page.genbank_pat link="https://www.ncbi.nlm.nih.gov/assembly/YOURACCHERE" %} </td>
           </tr>
           <tr><td class="label">ENA</td>
               <td> {% include resource-links.html data=page.genbank_mat link="https://www.ebi.ac.uk/ena/browser/view/YOURACCHERE" %} </td>
               <td> {% include resource-links.html data=page.genbank_pat link="https://www.ebi.ac.uk/ena/browser/view/YOURACCHERE" %} </td>
           </tr>
           <tr><td class="label">UCSC Genome Browser</td>
               <td> {% include resource-links.html data=page.genbank_mat link="https://genome.ucsc.edu/h/YOURACCHERE" %} </td>
               <td> {% include resource-links.html data=page.genbank_pat link="https://genome.ucsc.edu/h/YOURACCHERE" %} </td>
           </tr></tbody>
    </table>
  {% endif %}
</div class="sd-info">

<div class="sd-image">
  <div style="display: inline; margin: 0;">
    <div style="background-image:url({{ image }}); " class="species-image-full" src="">
    </div>
    <div class="species-image-lic">
      {{ license }}
    </div>
  </div>
</div>
</div>
</div class="section">

{% include section-divider.html title="Raw Data" %}

{% unless page.last_raw_data %}

<div class="section">
<div class="data-use">
<p style="text-align: center;">
<em style="color:maroon">No raw data.</em>
</p>
</div>
</div>

{% else %}

<div class="section">
<table class="raw-data-table">
<thead><tr><th rowspan="2" class="label">Datatype</th>
           <th colspan="2">Approximate Size</th>
           <th rowspan="2">Download Size</th>
       </tr>
       <tr><th>Bases</th>
           <th>Coverage</th>
       </tr></thead>
<tbody>
  {% if page.data_10x_bytes.size             > 0 %} <tr><td class="label">10x                       </td><td>{{ page.data_10x_bases             }}</td><td>{{ page.data_10x_coverage             }}</td><td>{{ page.data_10x_bytes             }}</td></tr> {% endif %}
  {% if page.data_arima_bytes.size           > 0 %} <tr><td class="label">Arima                     </td><td>{{ page.data_arima_bases           }}</td><td>{{ page.data_arima_coverage           }}</td><td>{{ page.data_arima_bytes           }}</td></tr> {% endif %}
  {% if page.data_bionano_bytes.size         > 0 %} <tr><td class="label">Bionano                   </td><td>{{ page.data_bionano_bases         }}</td><td>{{ page.data_bionano_coverage         }}</td><td>{{ page.data_bionano_bytes         }}</td></tr> {% endif %}
  {% if page.data_dovetail_bytes.size        > 0 %} <tr><td class="label">Dovetail Genomics         </td><td>{{ page.data_dovetail_bases        }}</td><td>{{ page.data_dovetail_coverage        }}</td><td>{{ page.data_dovetail_bytes        }}</td></tr> {% endif %}
  {% if page.data_illumina_bytes.size        > 0 %} <tr><td class="label">Illumina                  </td><td>{{ page.data_illumina_bases        }}</td><td>{{ page.data_illumina_coverage        }}</td><td>{{ page.data_illumina_bytes        }}</td></tr> {% endif %}
  {% if page.data_ont_bytes.size             > 0 %} <tr><td class="label">Oxford Nanopore Simplex   </td><td>{{ page.data_ont_bases             }}</td><td>{{ page.data_ont_coverage             }}</td><td>{{ page.data_ont_bytes             }}</td></tr> {% endif %}
  {% if page.data_ontduplex_bytes.size       > 0 %} <tr><td class="label">Oxford Nanopore Duplex    </td><td>{{ page.data_ontduplex_bases       }}</td><td>{{ page.data_ontduplex_coverage       }}</td><td>{{ page.data_ontduplex_bytes       }}</td></tr> {% endif %}
  {% if page.data_pacbio_bytes.size          > 0 %} <tr><td class="label">PacBio CLR                </td><td>{{ page.data_pacbio_bases          }}</td><td>{{ page.data_pacbio_coverage          }}</td><td>{{ page.data_pacbio_bytes          }}</td></tr> {% endif %}
  {% if page.data_pacbiohifi_fqgz_bytes.size > 0 %} <tr><td class="label">PacBio HiFi (.fq, Q&ge;20)</td><td>{{ page.data_pacbiohifi_fqgz_bases }}</td><td>{{ page.data_pacbiohifi_fqgz_coverage }}</td><td>{{ page.data_pacbiohifi_fqgz_bytes }}</td></tr> {% endif %}
  {% if page.data_pacbiohifi_bam_bytes.size  > 0 %} <tr><td class="label">PacBio HiFi (.bam, Q&ge;0)</td><td>{{ page.data_pacbiohifi_bam_bases  }}</td><td>{{ page.data_pacbiohifi_bam_coverage  }}</td><td>{{ page.data_pacbiohifi_bam_bytes  }}</td></tr> {% endif %}
  {% if page.data_pacbiohifi_clr_bytes.size  > 0 %} <tr><td class="label">PacBio HiFi (subreads)    </td><td>{{ page.data_pacbiohifi_clr_bases  }}</td><td>{{ page.data_pacbiohifi_clr_coverage  }}</td><td>{{ page.data_pacbiohifi_clr_bytes  }}</td></tr> {% endif %}
  {% if page.data_phase_bytes.size           > 0 %} <tr><td class="label">Phase Genomics            </td><td>{{ page.data_phase_bases           }}</td><td>{{ page.data_phase_coverage           }}</td><td>{{ page.data_phase_bytes           }}</td></tr> {% endif %}
</tbody>
</table>

<p class="raw-data-size">  <!---<div align="right">--->
  {% if page.genome_size_display %}
  Coverage based on genome size {{ page.genome_size_display }}.
  {% else %}
  No genome size estimate available.
  {% endif %}
</p>

<p class="raw-data-size">  <!---<div align="right">--->
  Last upload on {{ page.last_raw_data | date: "%-d %B %Y" }}.
</p>

<div>
<p>Download each datatype using the <A HREF="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html">AWS Command Line Interface</A>:</p>
<div class="raw-data-links">
  {% if page.data_10x_links             %} {{ page.data_10x_links             }} {% endif %}
  {% if page.data_arima_links           %} {{ page.data_arima_links           }} {% endif %}
  {% if page.data_bionano_links         %} {{ page.data_bionano_links         }} {% endif %}
  {% if page.data_dovetail_links        %} {{ page.data_dovetail_links        }} {% endif %}
  {% if page.data_illumina_links        %} {{ page.data_illumina_links        }} {% endif %}
  {% if page.data_ont_links             %} {{ page.data_ont_links             }} {% endif %}
  {% if page.data_ontduplex_links       %} {{ page.data_ontduplex_links       }} {% endif %}
  {% if page.data_pacbio_links          %} {{ page.data_pacbio_links          }} {% endif %}
  {% if page.data_pacbiohifi_fqgz_links %} {{ page.data_pacbiohifi_fqgz_links }} {% endif %}
  {% if page.data_pacbiohifi_bam_links  %} {{ page.data_pacbiohifi_bam_links  }} {% endif %}
  {% if page.data_pacbiohifi_clr_links  %} {{ page.data_pacbiohifi_clr_links  }} {% endif %}
  {% if page.data_phase_links           %} {{ page.data_phase_links           }} {% endif %}
</div class="raw-data-links">
</div>
</div class="section">

{% endunless %}


<!-- NOTES --->

{% if page.notes %}
{% include section-divider.html title="Notes" %}
<div class="notes">
<em style="color:red">{{ page.notes }}</em><br/>
</div>
{% endif %}

<!-- ASSEMBLIES --->

{% include section-divider.html title="Assemblies" %}

{% comment %}
  Create an assembly-stats block for any assembly that exists for up to six individuals.
  A block will have up to two assemblies shown (with an optional mito sequence):
    PRI/ALT
    PAT/MAT (with optional merged version)
    DIP
  An individual shouldn't have both PRI/ALT and PAT/MAT, but it will still
  display properly (except that the mito will be shown for both).

  There's a LOT of baloney here to get a divider between individuals shown, but that
  same baloney lets us decide early if there are any assemblies at all, and if there are none,
  to show an entirely different block.
{% endcomment %}

{% if page.pri1seq or page.alt1seq or page.mat1seq or page.pat1seq or page.mgd1seq or page.dip1seq or page.mito1seq %} {% assign d1 = true %} {% else %} {% assign d1 = false %} {% endif %}
{% if page.pri2seq or page.alt2seq or page.mat2seq or page.pat2seq or page.mgd2seq or page.dip2seq or page.mito2seq %} {% assign d2 = true %} {% else %} {% assign d2 = false %} {% endif %}
{% if page.pri3seq or page.alt3seq or page.mat3seq or page.pat3seq or page.mgd3seq or page.dip3seq or page.mito3seq %} {% assign d3 = true %} {% else %} {% assign d3 = false %} {% endif %}
{% if page.pri4seq or page.alt4seq or page.mat4seq or page.pat4seq or page.mgd4seq or page.dip4seq or page.mito4seq %} {% assign d4 = true %} {% else %} {% assign d4 = false %} {% endif %}
{% if page.pri5seq or page.alt5seq or page.mat5seq or page.pat5seq or page.mgd5seq or page.dip5seq or page.mito5seq %} {% assign d5 = true %} {% else %} {% assign d5 = false %} {% endif %}
{% if page.pri6seq or page.alt6seq or page.mat6seq or page.pat6seq or page.mgd6seq or page.dip6seq or page.mito6seq %} {% assign d6 = true %} {% else %} {% assign d6 = false %} {% endif %}

{% if d1 %}                         {% assign div12a = true %} {% else %} {% assign div12a = false %} {% endif %}
{% if d1 or d2 %}                   {% assign div23a = true %} {% else %} {% assign div23a = false %} {% endif %}
{% if d1 or d2 or d3 %}             {% assign div34a = true %} {% else %} {% assign div34a = false %} {% endif %}
{% if d1 or d2 or d3 or d4 %}       {% assign div45a = true %} {% else %} {% assign div45a = false %} {% endif %}
{% if d1 or d2 or d3 or d4 or d5 %} {% assign div56a = true %} {% else %} {% assign div56a = false %} {% endif %}

{% if d2 or d3 or d4 or d5 or d6 %} {% assign div12b = true %} {% else %} {% assign div12b = false %} {% endif %}
{% if d3 or d4 or d5 or d6 %}       {% assign div23b = true %} {% else %} {% assign div23b = false %} {% endif %}
{% if d4 or d5 or d6 %}             {% assign div34b = true %} {% else %} {% assign div34b = false %} {% endif %}
{% if d5 or d6 %}                   {% assign div45b = true %} {% else %} {% assign div45b = false %} {% endif %}
{% if d6 %}                         {% assign div56b = true %} {% else %} {% assign div56b = false %} {% endif %}


{% unless d1 or d2 or d3 or d4 or d5 or d6 %}

<div class="section">
<div class="data-use">
<p style="text-align: center;">
<em style="color:maroon">No assemblies.</em>
</p>
</div>
</div>

{% else %}

<div class="section">
<div class="assemblies">

{% assign shown = false %}

{% include assembly-stats.html individual=1 genome_size_display=page.genome_size_display
     llabel="Primary"     lseq=page.pri1seq  lversion=page.pri1version  ldate=page.pri1date  lfilesize=page.pri1filesize  lsizes=page.pri1sizes
     rlabel="Alternate"   rseq=page.alt1seq  rversion=page.alt1version  rdate=page.alt1date  rfilesize=page.alt1filesize  rsizes=page.alt1sizes %}
{% include assembly-stats.html individual=1 genome_size_display=page.genome_size_display
     llabel="Maternal"    lseq=page.mat1seq  lversion=page.mat1version  ldate=page.mat1date  lfilesize=page.mat1filesize  lsizes=page.mat1sizes
     rlabel="Paternal"    rseq=page.pat1seq  rversion=page.pat1version  rdate=page.pat1date  rfilesize=page.pat1filesize  rsizes=page.pat1sizes
     tlabel="Merged Trio" tseq=page.mgd1seq  tversion=page.mgd1version  tdate=page.mgd1date  tfilesize=page.mgd1filesize  tsizes=page.mgd1sizes %}
{% include assembly-stats.html individual=1 genome_size_display=page.genome_size_display
     tlabel="Diploid"     tseq=page.dip1seq  tversion=page.dip1version  tdate=page.dip1date  tfilesize=page.dip1filesize  tsizes=page.dip1sizes %}
{% include assembly-stats.html individual=1
                          mseq=page.mito1seq mversion=page.mito1version mdate=page.mito1date mfilesize=page.mito1filesize %}

{% if d1 %} {% assign shown = false %} {% endif %}
{% if shown == false and d1 and div12b %}
{% assign shown = true %} <br><hr><br>
{% endif %}


{% include assembly-stats.html individual=2 genome_size_display=page.genome_size_display
     llabel="Primary"     lseq=page.pri2seq  lversion=page.pri2version  ldate=page.pri2date  lfilesize=page.pri2filesize  lsizes=page.pri2sizes
     rlabel="Alternate"   rseq=page.alt2seq  rversion=page.alt2version  rdate=page.alt2date  rfilesize=page.alt2filesize  rsizes=page.alt2sizes %}
{% include assembly-stats.html individual=2 genome_size_display=page.genome_size_display
     llabel="Maternal"    lseq=page.mat2seq  lversion=page.mat2version  ldate=page.mat2date  lfilesize=page.mat2filesize  lsizes=page.mat2sizes
     rlabel="Paternal"    rseq=page.pat2seq  rversion=page.pat2version  rdate=page.pat2date  rfilesize=page.pat2filesize  rsizes=page.pat2sizes
     tlabel="Merged Trio" tseq=page.mgd2seq  tversion=page.mgd2version  tdate=page.mgd2date  tfilesize=page.mgd2filesize  tsizes=page.mgd2sizes %}
{% include assembly-stats.html individual=2 genome_size_display=page.genome_size_display
     tlabel="Diploid"     tseq=page.dip2seq  tversion=page.dip2version  tdate=page.dip2date  tfilesize=page.dip2filesize  tsizes=page.dip2sizes %}
{% include assembly-stats.html individual=2
                          mseq=page.mito2seq mversion=page.mito2version mdate=page.mito2date mfilesize=page.mito2filesize %}

{% if d2 %} {% assign shown = false %} {% endif %}
{% if shown == false and d2 and div23b %}
{% assign shown = true %} <br><hr><br>
{% endif %}


{% include assembly-stats.html individual=3 genome_size_display=page.genome_size_display
     llabel="Primary"     lseq=page.pri3seq  lversion=page.pri3version  ldate=page.pri3date  lfilesize=page.pri3filesize  lsizes=page.pri3sizes
     rlabel="Alternate"   rseq=page.alt3seq  rversion=page.alt3version  rdate=page.alt3date  rfilesize=page.alt3filesize  rsizes=page.alt3sizes %}
{% include assembly-stats.html individual=3 genome_size_display=page.genome_size_display
     llabel="Maternal"    lseq=page.mat3seq  lversion=page.mat3version  ldate=page.mat3date  lfilesize=page.mat3filesize  lsizes=page.mat3sizes
     rlabel="Paternal"    rseq=page.pat3seq  rversion=page.pat3version  rdate=page.pat3date  rfilesize=page.pat3filesize  rsizes=page.pat3sizes
     tlabel="Merged Trio" tseq=page.mgd3seq  tversion=page.mgd3version  tdate=page.mgd3date  tfilesize=page.mgd3filesize  tsizes=page.mgd3sizes %}
{% include assembly-stats.html individual=3 genome_size_display=page.genome_size_display
     tlabel="Diploid"     tseq=page.dip3seq  tversion=page.dip3version  tdate=page.dip3date  tfilesize=page.dip3filesize  tsizes=page.dip3sizes %}
{% include assembly-stats.html individual=3
                          mseq=page.mito3seq mversion=page.mito3version mdate=page.mito3date mfilesize=page.mito3filesize %}

{% if d3 %} {% assign shown = false %} {% endif %}
{% if shown == false and d3 and div34b %}
{% assign shown = true %} <br><hr><br>
{% endif %}


{% include assembly-stats.html individual=4 genome_size_display=page.genome_size_display
     llabel="Primary"     lseq=page.pri4seq  lversion=page.pri4version  ldate=page.pri4date  lfilesize=page.pri4filesize  lsizes=page.pri4sizes
     rlabel="Alternate"   rseq=page.alt4seq  rversion=page.alt4version  rdate=page.alt4date  rfilesize=page.alt4filesize  rsizes=page.alt4sizes %}
{% include assembly-stats.html individual=4 genome_size_display=page.genome_size_display
     llabel="Maternal"    lseq=page.mat4seq  lversion=page.mat4version  ldate=page.mat4date  lfilesize=page.mat4filesize  lsizes=page.mat4sizes
     rlabel="Paternal"    rseq=page.pat4seq  rversion=page.pat4version  rdate=page.pat4date  rfilesize=page.pat4filesize  rsizes=page.pat4sizes
     tlabel="Merged Trio" tseq=page.mgd4seq  tversion=page.mgd4version  tdate=page.mgd4date  tfilesize=page.mgd4filesize  tsizes=page.mgd4sizes %}
{% include assembly-stats.html individual=4 genome_size_display=page.genome_size_display
     tlabel="Diploid"     tseq=page.dip4seq  tversion=page.dip4version  tdate=page.dip4date  tfilesize=page.dip4filesize  tsizes=page.dip4sizes %}
{% include assembly-stats.html individual=4
                          mseq=page.mito4seq mversion=page.mito4version mdate=page.mito4date mfilesize=page.mito4filesize %}

{% if d4 %} {% assign shown = false %} {% endif %}
{% if shown == false and d4 and div45b %}
{% assign shown = true %} <br><hr><br>
{% endif %}


{% include assembly-stats.html individual=5 genome_size_display=page.genome_size_display
     llabel="Primary"     lseq=page.pri5seq  lversion=page.pri5version  ldate=page.pri5date  lfilesize=page.pri5filesize  lsizes=page.pri5sizes
     rlabel="Alternate"   rseq=page.alt5seq  rversion=page.alt5version  rdate=page.alt5date  rfilesize=page.alt5filesize  rsizes=page.alt5sizes %}
{% include assembly-stats.html individual=5 genome_size_display=page.genome_size_display
     llabel="Maternal"    lseq=page.mat5seq  lversion=page.mat5version  ldate=page.mat5date  lfilesize=page.mat5filesize  lsizes=page.mat5sizes
     rlabel="Paternal"    rseq=page.pat5seq  rversion=page.pat5version  rdate=page.pat5date  rfilesize=page.pat5filesize  rsizes=page.pat5sizes
     tlabel="Merged Trio" tseq=page.mgd5seq  tversion=page.mgd5version  tdate=page.mgd5date  tfilesize=page.mgd5filesize  tsizes=page.mgd5sizes %}
{% include assembly-stats.html individual=5 genome_size_display=page.genome_size_display
     tlabel="Diploid"     tseq=page.dip5seq  tversion=page.dip5version  tdate=page.dip5date  tfilesize=page.dip5filesize  tsizes=page.dip5sizes %}
{% include assembly-stats.html individual=5
                          mseq=page.mito5seq mversion=page.mito5version mdate=page.mito5date mfilesize=page.mito5filesize %}

{% if d5 %} {% assign shown = false %} {% endif %}
{% if shown == false and d5 and div56b %}
{% assign shown = true %} <br><hr><br>
{% endif %}


{% include assembly-stats.html individual=6 genome_size_display=page.genome_size_display
     llabel="Primary"     lseq=page.pri6seq  lversion=page.pri6version  ldate=page.pri6date  lfilesize=page.pri6filesize  lsizes=page.pri6sizes
     rlabel="Alternate"   rseq=page.alt6seq  rversion=page.alt6version  rdate=page.alt6date  rfilesize=page.alt6filesize  rsizes=page.alt6sizes %}
{% include assembly-stats.html individual=6 genome_size_display=page.genome_size_display
     llabel="Maternal"    lseq=page.mat6seq  lversion=page.mat6version  ldate=page.mat6date  lfilesize=page.mat6filesize  lsizes=page.mat6sizes
     rlabel="Paternal"    rseq=page.pat6seq  rversion=page.pat6version  rdate=page.pat6date  rfilesize=page.pat6filesize  rsizes=page.pat6sizes
     tlabel="Merged Trio" tseq=page.mgd6seq  tversion=page.mgd6version  tdate=page.mgd6date  tfilesize=page.mgd6filesize  tsizes=page.mgd6sizes %}
{% include assembly-stats.html individual=6 genome_size_display=page.genome_size_display
     tlabel="Diploid"     tseq=page.dip6seq  tversion=page.dip6version  tdate=page.dip6date  tfilesize=page.dip6filesize  tsizes=page.dip6sizes %}
{% include assembly-stats.html individual=6
                          mseq=page.mito6seq mversion=page.mito6version mdate=page.mito6date mfilesize=page.mito6filesize %}

</div>
</div>
{% endunless %}


<!-- DATA USE POLICY --->

{% include section-divider.html title="Data Use Policy" %}

<div class="section">
<div class="data-use">
<p>
Samples and data come from a variety of sources. To support fair and productive use of this data,
please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/"><em>Data Use Policy</em></a>
and contact us with any questions.
</p>
</div>
</div>
